jekyll/_cci2/vs-code-extension-quickstart.adoc

```diff
@@ -0,0 +1,117 @@
+
+There are two ways to install the CircleCI extension:
+
+* In VS Code, click the **Extensions** icon on the left-hand Activity Bar, or open **View > Extensions**.
+
+* Or, download the extension from the link:https://marketplace.visualstudio.com/items?itemName=circleci.circleci[VS Code Marketplace].
+
+[#authenticate-into-circleci]
+== 2. Authenticate into CircleCI
+
+. In VS Code, open the CircleCI panel by clicking on the CircleCI icon on the activity bar, then click “Log in”.
+
+. You will be prompted to enter a xref:managing-api-tokens#overview[personal API token]. You may create one in your CircleCI link:https://app.circleci.com/settings/user/tokens[**User Settings > Personal API Tokens**.]
++
+**CircleCI server**: Make sure to check the "I'm a CircleCI Server customer," then enter your self-hosted server URL.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_login.png[Log in to CircleCI from VS Code]
+
+[#set-up-your-project-in-vs-code]
+== 3. Set up your project in VS Code
+
+NOTE: The pipelines manager does not yet support GitLab projects.
+
+If the root of your workspace is a CircleCI project or contains a project that you follow, your project will be detected automatically, and the pipelines panel will be populated with your most recent pipelines.
+
+Otherwise, you may manually select a project by opening the extension's settings either through the VS Code command `CircleCI: Open Settings`, or by clicking on the settings icon (⚙️) at the top of the pipelines panel.
+
+NOTE: Support for link:https://code.visualstudio.com/docs/editor/multi-root-workspaces[multi-root workspaces] will be available soon.
+
+[#configure-the-pipelines-panel-in-vs-code]
+== 4. Configure the pipelines panel in VS Code
+
+By default, the pipelines panel displays all the latest pipelines run on your project. However, you can also configure it to show only the pipelines you are interested in. You can do so in the extension's settings page (run the command `CircleCI: Open Settings`, or click on the settings icon (⚙️) at the top of the pipelines panel).
+
+Specifically, you can choose to display:
+
+* All pipelines triggered by any user (author), or only pipelines you trigger.
+* All branches, only your current branch, or a specific branch.
+** You can also choose to “Always display default branch.”
+* Only pipelines that include workflows and jobs with specific statuses.
+
+[#configure-notifications]
+== 5. Configure notifications
+
+By default, you will receive VS Code pop-up notifications every time one of the workflows in your pipelines panel enters a failing status, or needs your approval.
+
+You can configure specific workflow status changes you want to be notified for in the Settings page.
+
+[#configure-ssh-keys]
+== 6. Configure SSH keys
+
+The extension allows you to xref:ssh-access-jobs#[rerun jobs with SSH] directly from within VS Code, either in the terminal or in a link:https://code.visualstudio.com/docs/remote/ssh[remote VS Code window].
```

Comment:
I think this section needs marking out as not available to GitLab users too - SSH reruns not currently supported

Reply:
Given that Gitlab projects are not supported in general by the Pipelines Manager, no need to specify this. 
We should remember to do so when we add support for standalone projects